sandboxed_containers/sandboxed-containers-4.11-release-notes.adoc

```diff
@@ -26,7 +26,78 @@ This product is fully supported and enabled by default as of {product-title} 4.1
 [id="sandboxed-containers-1-3-known-issues"]
 == Known issues
 
-
+* If you are using {sandboxed-containers-first}, you might receive SELinux denials when accessing files or directories mounted from the `hostPath` volume in an {product-title} cluster. These denials can occur even when running privileged sandboxed containers because privileged sandboxed containers do not disable SELinux checks.
++
+Following SELinux policy on the host guarantees full isolation of the host file system from the sandboxed workload by default, and provides stronger protection against potential security flaws in the `virtiofsd` daemon or QEMU.
++
+If the mounted files or directories do not have specific SELinux requirements on the host, you can use local persistent volumes as an alternative. Files are automatically relabeled to `container_file_t`, following SELinux policy for container runtimes. See xref:../storage/persistent_storage/persistent-storage-local.adoc[Persistent storage using local volumes] for more information.
++
+Automatic relabeling is not an option when mounted files or directories are expected to have specific SELinux labels on the host. Instead, you can set custom SELinux rules on the host in order to allow the `virtiofsd` daemon to access these specific labels. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1904609[*BZ#1904609*])
+
+* The {sandboxed-containers-operator} pods that use container CPU resource limits to increase the number of available CPUs for the pod might receive fewer CPUs than requested. If the functionality is available inside the container, you can diagnose CPU resources by using `oc rsh <pod>` and running the `lscpu` command.
++
+[source,terminal]
+----
+$ lscpu
+----
++
+.Example output
++
+[source,terminal]
+----
+CPU(s):                          16
+On-line CPU(s) list:             0-12,14,15
+Off-line CPU(s) list:            13
+----
++
+The list of available offline CPUs will likely change from run to run in an unpredictable manner.
++
+As a workaround, you can use a pod annotation to request additional CPUs rather than setting a CPU limit. The method of allocating processors is different, and CPUs requested by means of pod annotation are not affected by this issue. Rather than setting a CPU limit, the following `annotation` must be added to the metadata of the pod:
++
+[source,yaml]
+----
+metadata:
+  annotations:
+    io.katacontainers.config.hypervisor.default_vcpus: "16"
+----
++
+(link:https://issues.redhat.com/browse/KATA-1376[*KATA-1376*])
+
+* The progress of the runtime installation is shown in the `status` section of the `kataConfig` CR. However, the progress is not shown if all of the following conditions are true:
+
+** The cluster has a machine config pool `worker` without any members (`machinecount=0`).
```

Comment:
You could avoid using parenthesis by expanding "(`machinecount=0`)" into a separate sentence. You could also describe where the `machinecount=0` value is seen, for readers who might not know where to look for that.

Reply:
Going to have to defer to dev on this.  I am not finding "machinecount" in any files I am familiar with. I think it isn in the MCO.  @jensfr would probably be able to answer.

Reply:
in the description of the Machine Config Pool, it can be viewed with 
``` oc describe mcp worker

...
  Degraded Machine Count:     0
  Machine Count:              3
  Observed Generation:        2
  Ready Machine Count:        3
  Unavailable Machine Count:  0
  Updated Machine Count:      3
Events:                       <none>`